root/admin/attributes/form.tt

```diff
@@ -34,6 +34,17 @@
         </p>
 
     [% ELSE %]
+        [%~ IF model == "CollectionType" || model == "SeriesType" ~%]
+            [% form_row_select(r, 'entity_type', add_colon(l('Entity type'))) %]
+            [%~ IF r.form.field('entity_type').value.size ~%]
```

Comment:
Thank you for having spotted this issue.
I replaced the faulty condition by a cleaner one on `c.action.name` in https://github.com/metabrainz/musicbrainz-server/pull/471/commits/cde0df93294647929c713687922ce70b55df3d01.